routes/degree_days.py

```diff
@@ -0,0 +1,375 @@
+            )
+        for tempstat in range(0, 3):
+            request_str = get_dd_wcps_request_str(
+                x, y, cov_id, "0:0", timestring, tempstat
+            )
+            point_data_list.append(
+                await fetch_data([generate_wcs_query_url(request_str)])
+            )
+
+    if horp in ["projected", "hp"]:
+        min_year = years_lu["projected"]["min"]
+        max_year = years_lu["projected"]["max"]
+        timestring = (
+            f'"{min_year}-01-01T00:00:00.000Z":"{max_year}-01-01T00:00:00.000Z"'
+        )
+        if start_year is not None:
+            timestring = (
+                f'"{start_year}-01-01T00:00:00.000Z":"{end_year}-01-01T00:00:00.000Z"'
+            )
+        for tempstat in range(0, 3):
+            request_str = get_dd_wcps_request_str(
+                x, y, cov_id, "1:2", timestring, tempstat
+            )
+            point_data_list.append(
+                await fetch_data([generate_wcs_query_url(request_str)])
+            )
+
+    return point_data_list
+
+
+def create_csv(data_pkg, var_ep, place_id=None, lat=None, lon=None):
+    """Create CSV file with metadata string and location based filename.
+    Args:
+        data_pkg (dict): JSON-like object of data
+        var_ep (str): heating, below_zero, thawing_index, or freezing_index
+        place_id: place identifier (e.g., AK124)
+        lat: latitude for points or None for polygons
+        lon: longitude for points or None for polygons
+    Returns:
+        CSV response object
+    """
+
+    fieldnames = [
+        "model",
+        "year",
+        "variable",
+        "value",
+    ]
+
+    csv_dicts = build_csv_dicts(
+        data_pkg,
+        fieldnames,
+    )
+
+    if var_ep == "heating":
+        metadata = (
+            "# dd is the total annual degree days below 65°F for the specified model\n"
+        )
+    elif var_ep == "below_zero":
+        metadata = (
+            "# dd is the total annual degree days below 0°F for the specified model\n"
+        )
+    elif var_ep == "thawing_index":
+        metadata = "# dd is the total annual degree days above freezing for the specified model\n"
+    elif var_ep == "freezing_index":
+        metadata = "# dd is the total annual degree days below freezing for the specified model\n"
+
+    filename = var_label_lu[var_ep] + " for " + lat + ", " + lon + ".csv"
+
+    return write_csv(csv_dicts, fieldnames, filename, metadata)
+
+
+@routes.route("/mmm/degree_days/<var_ep>/<horp>/<lat>/<lon>")
+@routes.route("/mmm/degree_days/<var_ep>/<horp>/<lat>/<lon>/<start_year>/<end_year>")
+def run_fetch_dd_point_data(var_ep, lat, lon, horp, start_year=None, end_year=None):
+    """Point data endpoint. Fetch point data for
+    specified lat/lon and return JSON-like dict.
+
+    Args:
+        var_ep (str): heating, below_zero, thawing_index, or freezing_index
+        lat (float): latitude
+        lon (float): longitude
+
```

Comment:
minor nitpick, add args for horp, start_year, end_year to docstring

Reply:
Fixed in 352c2027d213a687a5433fad95548ea0e6ba52fd.